### ISSUE TITLE
v2.0.2 in pypi is broken?

### ISSUE THREAD

**Issue by ayeowch** [ui/django-post_office#121]
v2.0.2 from https://pypi.python.org/pypi/django-post_office appears to be broken with the following error when running migrate:
```
KeyError: u"Migration pressroom.0002_auto_20150818_1135 dependencies reference nonexistent    parent node (u'post_office', u'0002_add_i18n_and_backend_alias')"
```
I had to revert to v2.0.1, i.e. pip install django-post-office==2.0.1, for now.

**Comment by foobacca** [comment]
Yes it appears that the 2.0.2 version has a load of the old south migrations in the migrations directory.  I think that means that django migrations considers that to be a south migrations directory and therefore ignores it.  You can manually delete all the extra migrations and it works - you want to delete all but 0001_initial and 0002__add_i18n_and_backend_alias

I haven't really worked out how the extra files would have ended up in the directory, but I imagine a clean rebuild would sort the issue out.

**Comment by mrmachine** [comment]
The wheel has the south migrations, but the tar.gz is OK.

**Comment by selwin** [comment]
Released 2.0.3 to fix the migration issues. Sorry about that.

**Comment by smcoll** [comment]
Hm this one tripped me up.  Is it possible to make a note in the docs or changelog regarding the issue?  Anyone installing 2.0.2 with the wheel is going to have this issue.
